tests/base.py

```diff
@@ -47,16 +47,17 @@ def get_type():
     def get_properties(self, original: bool = True):
         """Configuration properties required for the tap."""
         return_value = {
-            'start_date' : '2015-03-15T00:00:00Z',
             'account_id': os.getenv('TAP_FACEBOOK_ACCOUNT_ID'),
+            'start_date' : '2015-03-15T00:00:00Z',
             'end_date': '2015-03-16T00:00:00+00:00',
             'insights_buffer_days': '1'
         }
         if original:
             return return_value
 
+        # TODO sigh do we like really care about this though?? It just gets in the way while writing tests
```

Comment:
Seems fine to remove. I don't understand why it's there 🤷

Reply:
I think the original idea was to restrict how we change the start date in test_start_date.py to ensure our assertions were valid. But I think this is unnecessary, and gets in the way. I removed the comment in another in-progress PR.